app/src/main/java/com/github/bgabriel998/softwaredevproject/camera/CameraUiView.java

```diff
@@ -342,7 +342,7 @@ else if(POIPoints != null && !POIPoints.isEmpty()){
     private void drawPOIs(int actualDegree){
         //Go through all POIPoints
         for(POIPoint poiPoint : POIPoints){
-            if((int)poiPoint.getHorizontalBearing() == actualDegree){
+            if((int)poiPoint.getHorizontalBearing() == (actualDegree + 360) % 360){
```

Comment:
I have a doubt about the `+ 360` part. I understand that % can return a negative value if actualDegree < 0, so we need it. 
But if an angle can be negative, are we sure that it is still always greater than -360? Because if `actualDegree` were less than -360, it would return a negative number even after adding 360 to it.
What if instead of adding 360 we do something like this?
```suggestion
            if((int)poiPoint.getHorizontalBearing() == Math.abs(actualDegree) % 360){
```

Reply:
The name of ```actualDegree``` is probably a bit confusing. It is not the heading of the user but represents a direction of the compass.  The compass is basically a for-loop going from ```minDegrees``` to ```maxDegrees. ```actualDegree``` is the index of this for loop. ```minDegrees``` and ```maxDegrees``` are the heading of the user ```-+ the field of view of the camera divided by 2``` which corresponds to what the user can see. 
The heading of the user is calculated in ```compass.java```. This value is always between ```0°``` and ```360°```. This means that ```actualDegree``` will always be between ```-x``` and ```360 + x``` where ```x``` is the field of view of the camera divided by 2.
Regarding your commit, if the ```actualDegree``` would be ```-10°``` for example and you take the absolute value this would be ```10°```. However, ```-10°``` and ```10°``` are not the same. If you now do ```-10 + 360``` you get ```350°``` and ```-10°``` is equal to ```350°```.